docs/controller-skills/milkrun.md

```diff
@@ -0,0 +1,221 @@
+---
+title: Milk Run Monday
+---
+
+--8<-- "includes/abbreviations.md"
+
+Milk Run Monday is a very busy event, that tests the limits of controllers at it's busiest. It is important to understand some processes that makes the event run as smooth as possible for all pilot and controllers involved. The information on this page is extracted from various Local Instructions and Controller Skills pages, collated to give all of the most relevant information to controlling the Milk Run event.
+
+## ACD
+The job of the ACD controller during Milk Run is a fairly straight-forward one, however there are a couple of things to be mindful of.
+
+Ensure that all flight plans are correct, nothing like `DOSEL DCT RIVET`, `GPS DIRECT`, or anything else unusual like that, just stick to the pre-approved Jet and Non-Jet routes. This helps reduce workload on Enroute controllers down the line.
+
+### 16 Off Mode Departures at YMML
+A Reminder that [Off Mode Departures from Runway 16](../../aerodromes/classc/Melbourne/#off-mode-departures) during the 16A27D Runway Mode must be assigned the **ISPEG** SID, **NOT** the DOSEL SID. The **ISPEG** SID deconflicts concurrent 16 and 27 departures, whereas the DOSEL SID does not.
+
+## SMC
+### Standard Taxi Routes
+[YSSY](../../aerodromes/classc/Sydney/#standard-taxi-routes) and [YMML](../../aerodromes/classc/Melbourne/#standard-taxi-routes) both have Published Standard Taxi Routes that can be used to facilitate the orderly flow of traffic on the ground, but this does not preclude the SMC controller from using alternate routing if they deem it neccessary.
+
+### Queuing
+When giving taxi clearance, the SMC controller shall consider holding aircraft short of taxiways when in a long queue for a holding point, so that blockages are not caused on crossing taxiways. The Hotspots to pay the most attention to are circled below
+
+<figure markdown>
+![YMML Taxiway Hotspots](img/ymmlhotspots.png){ width="700" }
+  <figcaption>YMML Taxiway Hotspots</figcaption>
+</figure>
+
+<figure markdown>
+![YSSY Taxiway Hotspots](img/yssyhotspots.png){ width="700" }
+  <figcaption>YSSY Taxiway Hotspots</figcaption>
+</figure>
+
+### Protecting Runway Exits
+Arriving aircraft *should* hold short of any crossing taxiways upon exiting the runway, but that doesn't mean they always will! Hold short instructions should be used to protect exit taxiways where practical.
+
+#### YMML
+In order to protect common Runway Exit Taxiways, Consider instructing aircraft taxiing:  
+- Northbound on Taxiway `V`, to Hold Short of Taxiways `F` and `E`  
+- Southbound on Taxiway `V`, to Hold Short of Taxiways `G` and `J`
+
+#### YSSY
+In order to protect common Runway Exit Taxiways, Consider instructing aircraft taxiing:  
+- Northbound on Taxiway `B`, to Hold Short of Taxiway `B9` *(34 in use)*, or Taxiway `B7` *(16 in use)*  
+- Southbound on Taxiway `A`, to Hold Short of Taxiway `A2` *(34 in use)*, or Taxiway `A4` *(16 in use)*
```

Comment:
Maybe just a quick note that by doing so, you open up the use of J and Y as an alternate taxi route (because otherwise its easy to get nose to nose down there in the 34 direction)